internal/storage/storageutil/control_client.go

```diff
@@ -18,20 +18,31 @@ import (
 	"context"
 	"fmt"
 	"os"
+	"time"
 
 	control "cloud.google.com/go/storage/control/apiv2"
 	"github.com/googleapis/gax-go/v2"
 	"github.com/googlecloudplatform/gcsfuse/v2/internal/logger"
 	"google.golang.org/api/option"
+	"google.golang.org/grpc/codes"
 )
 
 func storageControlClientRetryOptions(clientConfig *StorageClientConfig) []gax.CallOption {
-	return []gax.CallOption{gax.WithRetry(func() gax.Retryer {
-		return gax.OnErrorFunc(gax.Backoff{
-			Max:        clientConfig.MaxRetrySleep,
-			Multiplier: clientConfig.RetryMultiplier,
-		}, ShouldRetry)
-	})}
+	return []gax.CallOption{
```

Comment:
In go sdk for control client, CreateFolder and DeleteFolder is not retrybe https://github.com/googleapis/google-cloud-go/blob/main/storage/control/apiv2/storage_control_client.go#L76

Do we want to keep the same?

Reply:
yes I also had same doubt, these methods mght not be idempotent so should not be retried, I didnt change it to keep the previous behaviour same in case we added retry after seeing any retriable error